m4_sessions/1803-inning/group_8.md

```diff
@@ -1,13 +1,36 @@
-Group 8
+## Group 8
 
-Group Members:
+### Group Members: 
+Spencer Herms, Lee Chow, Nyssa Keller, Ophus Wong
 
-Name of Session:
+### Name of Session: 
+Tech in Medicine: Balancing Ethics and Cutting Edge Technology
 
-Session Description (Why should students come to this session):
+### Session Description (Why should students come to this session): 
+- Technology is rapidly changing every aspect of our lives. As developers, we all understand the value of leveraging this to make our lives easier - medicine is no exception. Given the intimate involvement and ethical implications in the industry, it is important to explore this topic due to its widespread impact on many different walks of life. 
 
-Session Outcomes (What will students learn/leave with?):
+### Session Outcomes (What will students learn/leave with?):
+- Broader understanding of how technology affects the medical field
+- Help students develop and understand their own opinion regarding medtech
+- Give students an idea of what they would be working as a developer in medtech
 
-Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+  Intro/Housekeeping: 
```

Comment:
Good outline--in week 3 planning session, narrow down your resources.  You might want to set up a slack channel where you can share resources and then students will have access to lots of different ideas.  

Get specific on the scenarios you are going to propose.  Maybe 4 and you can each take on a small group discussion?